src/AppComponents/Common/Reader/OsmMapReader.cpp

```diff
@@ -250,33 +347,37 @@ bool OsmMapReader::init(
 {
     using namespace Core::Common;
 
-    // Note: For further development in @ambrosys:os-matcher-amb#38
-    size_t const eachNth = 1;
+    // Simplify Track
+    // ToDo: With correct type handling and inheritance the boost::geometry::simplify can be used directly
+    typedef boost::geometry::model::d2::point_xy<double> xy;
+    float maxDistance = 1.0; // GPS granularity is 2m, allow half of it since it can be left and right
```

Comment:
We also need to add this distance to the search radius, because I want ensure that I find a street x meters to the left to the track point, and with the simplification the track point could move 1m far away, right? Or is there any more accurate theory on how far the boundary may drift when specifying `max_distance`, i.e. what is the maximum error then?

Reply:
No you are right in your assumption, the `maxDistance` is indeed the maxDistance between the original and simplification and it has to be in relationship with the search Radius.

Reply:
Done